test/test_mavlink_protocol.cpp

```diff
@@ -122,83 +122,88 @@ static void play_stream(struct Context &ctx, const struct sockaddr_in &addr)
         log_error("Camera not found.");
         return;
     }
-
-    log_info("Select Horizontal resolution (0 for default): ");
-    h = read_int();
-
-    log_info("Select Vertical resolution (0 for default): ");
-    v = read_int();
-
-    mavlink_message_t out_msg;
-    mavlink_msg_set_video_stream_settings_pack(GCS_SYSID, MAV_COMP_ID_ALL, &out_msg,
-                                               ctx.connected_camera_sysid, MAV_COMP_ID_CAMERA,
-                                               camera_id, 0, h, v, 0, 0, "");
-
-    send_camera_msg(ctx, addr, out_msg);
-
-    mavlink_msg_command_long_pack(GCS_SYSID, MAV_COMP_ID_ALL, &out_msg, ctx.connected_camera_sysid,
-                                  MAV_COMP_ID_CAMERA, MAV_CMD_REQUEST_VIDEO_STREAM_INFORMATION, 0,
-                                  camera_id, 1, 0, 0, 0, 0, 0);
-
-    send_camera_msg(ctx, addr, out_msg);
 }
 
 static void handle_mavlink_message(struct Context &ctx, const struct sockaddr_in &addr, mavlink_message_t &msg)
 {
-    if (msg.compid != MAV_COMP_ID_CAMERA)
-        return;
+      //  log_info("MSGID=:%d and COMPID=:%d",msg.msgid,msg.compid);   
+    if (msg.compid < MAV_COMP_ID_CAMERA2){
```

Comment:
Why do you restrict to deal with only camera 1 ? Let it deal with any camera isn't it ?